script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_1.txt

```diff
@@ -159,7 +159,7 @@ Some students even call her the real student council president, because she is m
 It goes without saying that when I ran into trouble last week, she helped me out.
 <Page43>
 ...It's actually a bit of a blur.
-I'm not sure if it's because I'm not a morning person, or because my head got foggy from the thought of going back to that home from seven years ago.
+As Senpai said, I'm not sure if it's because I'm not a morning person, or because my head got foggy from the thought of going back to that home from seven years ago.//先輩の言う通り、朝に弱いことプラス、七年ぶりの帰宅で頭が曇っていたとしか思えない。
```

Comment:
Hmm, does the プラス not make this a "I'm not a morning person **and** I'm returning home for the first time in seven years, so my head is a little foggy"?

Reply:
Yes, you're right. And the "I'm not sure if" part is also wrong. Didn't even notice that.
"It must be because I'm not a morning person, as Senpai said, and because I'm returning home for the first time in seven years, so my head is a little foggy"

Reply:
The 言う通り is a bit weird, since she doesn't say anything in the lines above about him being a morning person, the only line it could reference is currently 
> "Look. It's me, it's me. I met you just last week, have you forgotten about me already?"

This is perhaps a little liberal, but reads better to me:
> It's just as Senpai[she] says. I'm not a morning person, and on top of that my thoughts are clouded by the prospect of returning home for the first time in seven years, so that must be why I didn't make the connection.

Reply:
She does say something "But it's true what they say about not being good in the morning, right?" line 141.
The "so that must be why I didn't make the connection" isn't needed. And the しか思えない thing that's not entirely there (I can think only of...).
>It's just as Senpai said. It must be because I'm not a morning person, and on top of that my thoughts are clouded by the prospect of returning home for the first time in seven years.

Reply:
Oh, yeah never mind. I totally missed that line.